query_test.go

```diff
@@ -93,32 +91,26 @@ func TestQueryWatch(t *testing.T) {
 		t.Fatal("expected 0 peers")
 	}
 
-	// Start servers one at a time in the background.
-	go func() {
-		for i := 1; i <= nrPeers; i++ {
-			s, err := NewServer(etcd, ServerCfg{Namespace: namespace})
-			if err != nil {
-				t.Fatal(err)
-			}
+	for i := 1; i <= nrPeers; i++ {
+		s, err := NewServer(etcd, ServerCfg{Namespace: namespace})
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		lis, err := net.Listen("tcp", "localhost:0")
+		if err != nil {
+			t.Fatal(err)
+		}
 
-			lis, err := net.Listen("tcp", "localhost:0")
+		done := make(chan error, 1)
```

Comment:
`done` is never used, right? We should remove it.

Reply:
Is it not used in tests anymore?